dev-python/wcwidth/wcwidth-0.2.5-r1.ebuild

```diff
@@ -24,13 +24,21 @@ RDEPEND="
 
 distutils_enable_tests pytest
 
-DOCS=()
-
 src_prepare() {
 	sed -e 's:--cov-append::' \
 		-e 's:--cov-report=html::' \
 		-e 's:--cov=wcwidth::' \
 		-i tox.ini || die
 	sed -i -e 's:test_package_version:_&:' tests/test_core.py || die
+
+	# causes pytest to fail
+	sed -i '/^looponfailroots =/d' tox.ini || die
```

Comment:
Bug ref

Reply:
Yeah, makes sence, I've added the reference